exasol_advanced_analytics_framework/event_handler/event_handler_result.py

```diff
@@ -2,9 +2,14 @@
 from typing import Optional, List, Dict, Any
 
 
+@dataclass()
+class EventHandlerReturnQuery:
+    query: str
+    query_columns: List[str]
```

Comment:
Please, use these classes, they can handle quoting and also validate for valid identifier

https://github.com/exasol/data-science-utils-python/tree/main/exasol_data_science_utils_python/preprocessing/sql/schema